medusa/search/manual.py

```diff
@@ -300,8 +300,8 @@ def get_provider_cache_results(series_obj, show_all_results=None, perform_search
             i['pretty_size'] = pretty_file_size(i['size']) if i['size'] > -1 else 'N/A'
             i['seeders'] = i['seeders'] if i['seeders'] >= 0 else '-'
             i['leechers'] = i['leechers'] if i['leechers'] >= 0 else '-'
-            i['pubdate'] = sbdatetime.convert_to_setting(parser.parse(i['pubdate'])).strftime(
-                app.DATE_PRESET + ' ' + app.TIME_PRESET) if i['pubdate'] else '-'
+            i['pubdate'] = parser.parse(i['pubdate']).strftime(
+                '%Y-%m-%d %H:%M:%S') if i['pubdate'] else '-'
```

Comment:
not using the `convert_to_setting`, it won't convert the datetime to user local timezone.

`added` is already in user local timezone, so don't need to be converted

Reply:
https://github.com/pymedusa/Medusa/blob/be284322b5b0ed8cf8624a3827f3210501573335/medusa/sbdatetime.py#L110-L117

That bit of code means if you select "network" as your timezone in config, you get 'published' times in an incorrect timezone. That's not the correct solution in my opinion.